### ISSUE TITLE
Error while loading Sveltia CMS start page

### ISSUE THREAD

**Issue by trustedtomato** [sveltia/sveltia-cms#165]
With the update to 0.34.4 or 0.34.5, I'm getting the following error when loading up the admin site:
![image](https://github.com/sveltia/sveltia-cms/assets/18383428/ebd277b7-c3fb-4327-bfbf-6b2e1d2d6385)
![image](https://github.com/sveltia/sveltia-cms/assets/18383428/92a933c8-a70e-4b43-9d35-a5db1967a998)

0.34.3 still works:
![image](https://github.com/sveltia/sveltia-cms/assets/18383428/f6dc9524-cbac-4bc1-8d71-398483465900)

My config.yml: http://udvarfeszt.hu/admin/config.yml

I'm guessing the YAML parser update in 0.34.4 went south, but I'm not sure yet.

**Comment by trustedtomato** [comment]
Seems like the config.yml can be reduced to just:
```yaml
backend:
  name: github
  repo: test/test

media_folder: 'images'

collections:
  - name: 'sections'
    label: 'Sections'
    files: []
```
and it still doesn't work. Also, I have error in the latest version of both Firefox and Chromium.

**Comment by kyoshino** [comment]
Hmm, will check.

**Comment by kyoshino** [comment]
Just shipped [v0.34.6](https://github.com/sveltia/sveltia-cms/releases/tag/v0.34.6) with the fix. Sorry about that 🙇🏼
